app/src/main/java/com/limelight/preferences/StreamSettings.java

```diff
@@ -31,11 +32,12 @@
 import com.limelight.PcView;
 import com.limelight.R;
 import com.limelight.binding.video.MediaCodecHelper;
+import com.limelight.utils.AspectRatioConverter;
 import com.limelight.utils.Dialog;
 import com.limelight.utils.UiHelper;
 
 import java.lang.reflect.Method;
-import java.util.Arrays;
+import java.util.*;
```

Comment:
I know wildcard imports are used in a few other places, but I'm not sure it's a good practice.